cluster/images/tools.cmds

```diff
@@ -24,6 +28,17 @@ sudo chmod a+x /opt/tools/gridss/2.7.2/gridss.sh
 sudo chmod a+x /opt/tools/gridss/2.8.3/gridss.sh
 sudo chmod a+x /opt/tools/gridss/2.9.3/gridss.sh
 sudo chmod a+x /opt/tools/gridss/2.9.4/gridss.sh
+
```

Comment:
For gridss 2.11 we only needed gridss.sh and gridss_annotate_vcf_repeatmasker.sh.

Reply:
Ok removed everything else that is not used by VirusBreakend.